src/Adapter/MSTest.TestAdapter/SourceGeneration/SourceGeneratedFileOperations.cs

```diff
@@ -7,18 +7,19 @@
 using Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices;
 using Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.Interface;
 
+#pragma warning disable CS0618 // ignore FileOperations deprecation
```

Comment:
This shouldn't be needed. The warning shows in IDE only due to a Roslyn bug that's fixed.

Reply:
fixed